src/main/java/TextView.java

```diff
@@ -34,6 +38,13 @@ public void update(Observable o, Object arg)
 	 */
 	void drawBoard(Board board)
 	{
+		int count = 1;
```

Comment:
- [ ] Give `count` a name that describes what it is counting. Maybe `rowCount`? My understanding is that this count is only used for counting the rows.